components/cam/test/system/input_tests_master

```diff
@@ -111,7 +111,6 @@ sm340 TSM.sh f4c5dm fv2d_8tsk 9s
 sm351 TSM.sh f0.9c4m outfrq24h+1850-2005_cam4 2d
 sm352 TSM.sh f0.9c4m outfrq24h+1850_cam4 2d
 
-sm360 TSM.sh f1.9c5carmdusdm outfrq3s_carma 9s
```

Comment:
git review won't let me tag a line not impacted (At least I don't know how to), but if you go up to line 54 in the new version of this file you have `cb460 TCB.sh f1.9c4carmsuldh`, didn't you delete this test?  It might be good to grep for the string "carm" in this file, I think that will catch all the straggler lines that reference the above deleted tests but don't mention carma outright.

Reply:
Good catch! I still don't understand what anything in this part of the code is for, so I wasn't quite sure what to look for.